autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -212,3 +238,66 @@ export function updateCourseCard(index: number, courseCard: CourseCardOptions, t
     return dispatch(updateCourseCardSync(index, courseCard));
   };
 }
+
+export function clearCourseCards(): ClearCourseCardsAction {
+  return { type: CLEAR_COURSE_CARDS };
+}
+
+/**
+ * Helper function to fetch updated sections for a course card and keep selected sections
+ * @param courseCard course card to update
+ * @param term term to fetch sections for
+ */
+async function updateSectionsForCourseCard(
+  courseCard: SerializedCourseCardOptions,
+  term: string,
+): Promise<CourseCardOptions> {
+  return fetchCourseCardFrom(courseCard, term).then((newCourseCard) => {
+    // course is now updated with new sections, re-select sections from original courseCard
+
+
+    const selectedSections = new Set(courseCard.sections);
+
+    newCourseCard.sections.forEach((section) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
Instead of disabling the rule, we should be able to just set the `props` option to false in the config, since we're reassigning a property of the param, not the param itself